src/TLuaInterpreter.cpp

```diff
@@ -2477,6 +2477,28 @@ int TLuaInterpreter::closeMudlet(lua_State* L)
     return 0;
 }
 
+int TLuaInterpreter::saveProfile(lua_State* L)
+{
+    Host* pHost = TLuaInterpreter::luaInterpreterMap[L];
+    if (!pHost) {
+        lua_pushstring(L, QLatin1String("saveProfile: NULL Host pointer - something is wrong!").data());
+        lua_error(L);
+        return 2;
+    }
+
+    auto result = pHost->saveProfile();
+
+    if (result.first == true) {
+        lua_pushboolean(L, true);
+        return 1;
+    } else {
+        lua_pushnil(L);
+        QString msg = QLatin1String("saveProfile: ") + result.second;
```

Comment:
http://doc.qt.io/qt-5/qstring.html Look for `QT_USE_QSTRINGBUILDER`

You have to `#include <QStringBuilder>` and use `%` to get the perf benefit mentioned.